index.html

```diff
@@ -1345,7 +1345,7 @@ <h2>Definition of Roles</h2>
 			<div class="role-description">
 				<p>A <rref>landmark</rref> that contains mostly site-oriented content, rather than page-specific content.</p>
 				<p>Site-oriented content typically includes things such as the logo or identity of the site sponsor, and a site-specific search tool. A banner usually appears at the top of the page and typically spans the full width.</p>
-				<p>User agents SHOULD treat elements with the role of <code>banner</code> as navigational <a>landmarks</a>.</p>
+				<p><a>Assistive technologies</a> SHOULD enable users to quickly navigate to elements with role <code>banner</code>. Mainstream <a>user agents</a> SHOULD enable users to quickly navigate to <a>elements</a> that have an implicit role of <code>banner</code>.</p>
 				<!-- keep the following paragraphs synced with the similar paragraphs in #main and #contentinfo-->
 				<p>Within any <rref>document</rref> or <rref>application</rref>, the author SHOULD mark no more than one <a>element</a> with the <code>banner</code> <a>role</a>.</p>
```

Comment:
I know this is not related to this PR, but I can't resist comment ... Wow is this out of date language. I guess it's been a while since I read it. It does not jive with the definition of role application. And, we do not typically refer to an entire web page as a document in the rest of the spec.

```suggestion
				<p>Authors SHOULD mark no more than one <a>element</a> on a page with the <code>banner</code> <a>role</a>.</p>
```

Reply:
I agree that "on a page" is easier to understand than "within any document (role) or application (role)" in that sentence.
However, I'm not sure whether or not the note box that follows that sentence (in banner, contentinfo, and main) needs to be reworded as well, so I've opened issue https://github.com/w3c/aria/issues/1320 to look into this.
> NOTE
Because document and application elements can be nested in the DOM, they may have multiple banner elements as DOM descendants, assuming each of those is associated with different document nodes, either by a DOM nesting (e.g., document within document) or by use of the aria-owns attribute.